hieradata/integration.yaml

```diff
@@ -215,7 +215,7 @@ govuk::node::s_licensify_lb::enable_feed_console: true
 govuk::node::s_licensify_mongo::licensify_mongo_encrypted: false
 govuk::node::s_logging::compress_srv_logs_hour: '9'
 govuk::node::s_monitoring::offsite_backups: false
-govuk::node::s_mysql_master::s3_bucket_name: 'govuk-mysql-xtrabackups-integration'
+govuk::node::s_mysql_master::s3_bucket_name: 'govuk-mysql-xtrabackups-production'
```

Comment:
In Production the backup takes place from a different machine (backup-1) so it's actually a slightly different variable.